examples/collisionshapes.js

```diff
@@ -1,9 +1,12 @@
 // @ts-check
 
+// # How kaplay handles collisions with different shapes
```

Comment:
why a #

Reply:
i tend to put # in titles sorry will remove later